ci-operator/step-registry/configure-registry-storage/set-nutanix-object/configure-registry-storage-set-nutanix-object-commands.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will set image registry to use nutanix object storage. 
+# It need deploy against nutanix platform.
+# Since nutanix object service still not support terraform, so we will use fixed bucket accordint to OCP version. And will clean up data on nutanix web console periodically.
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/xiuwang-nutanix-object-cred
```

Comment:
The credential is using the same account as aws-qe, right?

Reply:
@jianlinliu No, this credentials is generated by nutanix object service, this one is my pre-added